### ISSUE TITLE
[CoE Starter Kit - QUESTION] - Governance - Archival Approval Not Deleted after 3 weeks

### ISSUE THREAD

**Issue by dksamo1986** [microsoft/coe-starter-kit#5671]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Hi,

I have seen that we have currently 3 canvas app that have not been deleted after the 3 weeks mark. I think two of them are apps on Teams. One of the user got this mail for the app in Teams:

"However it cannot be deleted for you automatically and will need to be cleaned up and deleted manually.
This can happen for a variety of reasons including it being part of a solution or having dependencies.

Here is the error message received from an attempt to delete automatically.
App can only be deleted by uninstalling the managed solution(s) containing the app."

He then proceeded to remove the tab from Teams that held the app. 

Is there anything else he should do and will they ever be deleted? 

In regards to the last one that is in an environment, I am not sure why it has not been deleted. 

Best regards,

Sabri

![Archival Approval Table](https://github.com/microsoft/coe-starter-kit/assets/122786332/95ace8a8-5f7a-4b16-a29e-c1a0e60d6c5d)

### What solution are you experiencing the issue with?

Governance

### What solution version are you using?

3.6

### What app or flow are you having the issue with?

Admin | Inactivity notifications v2 (Clean Up and Delete)

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
There isnt anything we can do to remove these programatically. You can excuse the environment from the processes though.

**Comment by dksamo1986** [comment]
Hi Jenefer,

Can I just delete those 3 rows from the Archival table? I am assuming that they will be added again in the process and hopefully this time, it will work?

Best regards,

Sabri

**Comment by Jenefer-Monroe** [comment]
Yes you can delete the records in the archive table and then you can excuse individual apps/flows or entire environments from the processes to prevent it from being asked again. 

Please see [Exempt environments from the inactivity notifications process](https://learn.microsoft.com/power-platform/guidance/coe/setup-archive-components#exempt-environments-from-the-inactivity-notifications-process)

**Comment by dksamo1986** [comment]
Okay - thanks Jenefer.

**Comment by Jenefer-Monroe** [comment]
You are welcome!
